DESIGN.md

```diff
@@ -103,8 +93,14 @@ ram_per_jruby = (512, 768, 1024) if total memory (4-7 GB, 8-16 GB, 16 GB+)
 ram_per_jruby_code_cache = 128
 ```
 
-PuppetServer jrubies are constrained based on both how many jrubies fit into unallocated memory and unallocated processors. PuppetServer memory is then set the amount of memory required for all jrubies.
+PuppetServer jrubies are constrained based on both how many jrubies fit into unallocated memory and unallocated processors (one per processor). PuppetServer memory is then set the amount of memory required for all jrubies.
```

Comment:
(one per processor) => (one jruby per processor)

required for all jrubies => required for the total calculated number of jrubies